### PR TITLE
Prepend label for sub band centre frequencies

### PR DESCRIPTION
**Pull request by gtrymore**
- add a label before each frequency value to make it easier to refer to a particular band for bands with sub-band centre 
  frequencies.  E.g., “S0” for the first s-band sub-band.  “X3” for the 4th x-band sub-band.
-the implementation is  generic for all bands. 
- if a band has more than one sub-band we add the label.  The label is made up of two parts:  <letter><index>.  The letter is 
  the capitalised band letter.  The index is an integer, starting at zero for the first sub-band.
- for bands that only have 1 centre frequency (L and U-bands), we don't have a label for that one centre frequency.

Screenshot 2020-10-02 at 11 38 38

Screenshot 2020-10-02 at 11 38 54

Screenshot 2020-10-02 at 11 39 12

Screenshot 2020-10-02 at 11 39 23

Screenshot 2020-10-02 at 11 39 43

Screenshot 2020-10-02 at 11 39 55

JIRA: [ MT-1651](https://skaafrica.atlassian.net/browse/MT-1651)

### CODE REVIEW

**Review by mmphego — APPROVED** [review]
Looks good to me.

**Review by amakhaba — APPROVED** [review]
LGTM

File: app/scheduler/scheduler-home.js [hunk 1]
```diff
@@ -155,10 +156,24 @@
                 var subBandKeys = Object.keys(subBandConfig);
                 subBandKeys.forEach(function(sub_band) {
                     if (subBandConfig[sub_band].sub_bands) {
-                        vm.subBandsMap[sub_band] = subBandConfig[sub_band].sub_bands;
-                        vm.defaultCentreFreqMap[sub_band] = subBandConfig[sub_band].default;
+                        var subBandList = subBandConfig[sub_band].sub_bands;
+                        var labelledSubBands = {};
+                        var count = 0;
+                        var testDict = {};
```
Comment (lanceWilliams) [inline app/scheduler/scheduler-home.js:162]: Suggest change 'testDict' to 'labelledDefaultSubBand'.

File: app/scheduler/scheduler-home.js [hunk 2]
```diff
@@ -351,12 +368,27 @@
             vm.setNarrowFrequency(defaultNarrowFreq, i);
         };
 
+        vm.updateFrequency = function(label, freq) {
+            vm.setFrequency(freq);
+            vm.displayFrequency = vm.updateDisplayOfFrequency(label, freq);
+        };
+
+        vm.updateDisplayOfFrequency = function(label, freq) {
```
Comment (lanceWilliams) [inline app/scheduler/scheduler-home.js:376]: Suggest change 'vm.updateDisplayOfFrequency'  to  'vm.generateDisplayFrequency'.

**Review by bngcebetsha — APPROVED** [review]

**Review by tockards — APPROVED** [review]

**Pull request merged**